### DISCUSSION TITLE
Dockerfile for data-importer

### DISCUSSION THREAD

**Discussion by diogotcorreia** [firefly-iii/firefly-iii#5955] [Firefly III] [answered]
Hi,

Is there a place where I can find the data-importer's Dockerfile?

I found out firefly-iii's repo on Azure here, https://dev.azure.com/Firefly-III/MainImage, but I can't seem to find the one for the importer anywhere.

Thanks

**Comment by JC5** [comment, marked as answer]
In the top breadcrumb bar, click Firefly III, then click ImportToolImage

**Reply by diogotcorreia** [reply]
I did not have permission to click "Firefly III", it just went to a 404 page. Maybe it's private or something?
But anyway, the direct link is https://dev.azure.com/Firefly-III/ImportToolImage, so that's fixed!
Thanks
